ts/features/lollipop/utils/fetch.ts

```diff
@@ -62,15 +62,20 @@ export const lollipopFetch = (
           inputUrl,
           originalUrl
         };
+
+        const signatureParams: Array<string> = [
+          "x-pagopa-lollipop-original-method",
+          "x-pagopa-lollipop-original-url"
+        ];
+
+        if (lollipopConfig.signBody) {
+          signatureParams.unshift("Content-Digest", "Content-Type");
+        }
```

Comment:
unshift would change the array content, what do you think instead of declaring it as 

```suggestion
        const signatureParams: Array<string> = [
          ...(lollipopConfig.signBody ? ["Content-Digest", "Content-Type"] : []),
          "x-pagopa-lollipop-original-method",
          "x-pagopa-lollipop-original-url"
        ];
```